### DISCUSSION TITLE
google login with built apk does not work

### DISCUSSION THREAD

**Discussion by Reiserro** [capawesome-team/capacitor-firebase#585] [Q&A]
Hello i have managed to build my app, in android studio the login works with a normal build. it also works on the mobile phone when i use the build here. but when i create an apk and run an internal test in the google console this same code does not work. is there anything special to consider here? 

i have copied the methods almost indentically from the documentation, also added the rgcfaIncludeGoogle in the variables.gradle and also adapted the respective capacitor.config.json and ts files.

it is definitely due to this method as it triggers the catch block in my cry catch 

```
async signInWithGoogle_Capacitor(): Promise<SignInResult> {
  // 1. create credentials on the native layer
  const result = await FirebaseAuthentication.signInWithGoogle();
  // 2. Sign in on the web layer using the id token
  const credential = GoogleAuthProvider.credential(result.credential?.idToken);
  const auth = getAuth();
  await signInWithCredential(auth, credential);

  /* return result.additionalUserInfo.isNewUser */

  return result
};
```

thanks!

**Comment by robingenz** [comment]
What's the error message?

**Reply by Reiserro** [reply]
sorry i meant an android app bundle. instead of an apk. i have now removed the try catch block. and tried to get an error message, but it is loading all the time due to the awiat. is there anything you need to be aware of? in the signInWithGoogle_Capacitor method? or is the app bundle built internally differently than a normal app build?

**Reply by robingenz** [reply]
Do you call `initializeAuth` with `persistence` set to `indexedDBLocalPersistence` (see [docs](https://github.com/capawesome-team/capacitor-firebase/blob/main/packages/authentication/docs/firebase-js-sdk.md#installation))?

**Reply by Reiserro** [reply]
i'm super sorry to have wasted your time, i would first like to thank you for the awesome package. i'm super new and this will be my first app. i just forgot to exchange the self-signed sha1 key with the one from the google play console. greetings from stuttgart

**Comment by Reiserro** [comment]
exchange the self-signed sha1 key with the one from the google play console
